Gemfile

```diff
@@ -34,6 +34,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails', '~> 5.0.0'
   gem 'pry-rails'
+  gem 'faker', :git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'
```

Comment:
Is `:git => 'https://github.com/faker-ruby/faker.git', :branch => 'master'` needed?

Reply:
I use it for data generation